avm/interface/src/particle_parameters.rs

```diff
@@ -20,25 +20,30 @@ use std::borrow::Cow;
 
 /// Represents parameters obtained from a particle.
 #[derive(Debug, Clone, Serialize, Deserialize)]
-pub struct ParticleParameters<'init_peer_id, 'particle_id> {
+pub struct ParticleParameters<'init_peer_id, 'particle_id, 'current_peer_id> {
```

Comment:
@mikevoronov I think we should use single lifetime here.

Reply:
Agree